babelsberg/constraintinterpreter.js

```diff
@@ -497,29 +531,29 @@ Object.subclass('Constraint', {
                 throw new Error('BUG: No constraintobjects were created.');
             }
             this._enabled = true;
-            var nBegin = performance.now();
+            this.constraintvariables.each(function(v) {v._resetIsSolveable();});
+            console.time('solve');
             this.solver.solve();
-            var nEnd = performance.now();
-            console.log("Time to Solve in enable with solver below:" + (nEnd - nBegin) + " ms");
-            console.log(this.solver);
+            var time = console.timeEnd('solve');
```

Comment:
console.timeEnd gibt keinen Wert zurück (undefined), daher steht auf der Konsole jetzt sowas wie "Time to solve in enable with solver Cassowary: undefined ms"